### PR TITLE
lint fixes & 2024 block removed

### PR DESCRIPTION
**Pull request by Caleb-Cohen**
# Description

Proof of concepts to fix the tests. Needs to be refined a bit. 

## Type of change

Please select everything applicable. Please, do not delete any lines.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] This change requires an update to testing

## Issue

- [ ] Is this related to a specific issue? If so, please specify:

# Checklist:

- [ ] This PR is up to date with the main branch, and merge conflicts have been resolved
- [ ] I have executed `npm run test` and all tests have passed successfully or I have included details within my PR on the failure.
- [ ] I have executed `npm run lint` and resolved any outstanding errors. Most issues can be solved by executing `npm run format`
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings

### CODE REVIEW

**Review by cblanken — APPROVED** [review]
👍 looks good

File: server/models/Event.js
Comment (AchrafRhariss) [inline server/models/Event.js:1]: Hello,
IMHO, we should think to rename the variable 'limitTo2023' to match the desired context. ([line link here](https://github.com/Together-100Devs/Together/blob/eeb3f377807b030f8c41d5a10ec347444c84244f/server/models/Event.js#L34C1-L34C11))
for example: 'limitTo2029' or better 'limitToEventMaxDate'

**Comment by intelagense** [comment]
Closes #454

File: server/models/Event.js
Comment (intelagense) [reply server/models/Event.js:1]: I agree and we can create a separate issue for this later just because its completely broken right now 😅

**Review by intelagense — APPROVED** [review]
LGTM 
Tested locally and everything seems to work still.

**Pull request merged**